app/controllers/api/v2/template_invocations_controller.rb

```diff
@@ -0,0 +1,46 @@
+module Api
+  module V2
+    class TemplateInvocationsController < ::Api::V2::BaseController
+      include ::Api::Version2
+      include ::Foreman::Renderer
+
+      before_action :find_job_invocation, :only => %w{template_invocations}
+
+      api :GET, '/job_invocations/:id/template_invocations',
```

Comment:
In nested resources, we would usually do `:job_invocation_id` instead of `:id`, as the `:id` usually refers to the resource of the controller itself (which is `TemplateInvocation` in this case`)

Reply:
This leads to a slightly confusing situation where the API description states it is `job_invocation_id`, but in the controller it is accessible as `params[:id]`